index.html

```diff
@@ -16,25 +16,315 @@
+          </p>
+        </div>
+
+        <div class="languages-list-container">
+          
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio3.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item">JavaScript</li>
+            <li class="langage-item">HTML</li>
+            <li class="langage-item">CSS</li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio3.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+
+        <div class="text-container">
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+    </ul>
+    </div>
+
+
+ 
+
+</section>
+
+<section id="about" class="about-me">
+  <div class="left-content">
+    <h2 class="title">About Myself</h2>
+    <p class="info">
+      Hello I’m a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience! If
+      you like what you see and have a project you need coded, don’t
+      hestiate to contact me.
+    </p>
+    <a href="#" class="connect">LET'S CONNECT</a>
+    <ul class="social-media">
+      <li>
```

Comment:
- [ ] As asked by the previous reviewer. Kindly make changes according, to implementing the figma design.
- Your design:
<img width="374" alt="Screenshot 2022-03-20 at 1 01 38 AM" src="https://user-images.githubusercontent.com/70269507/159139987-68a26019-a520-4d24-9178-03bb8e91a5c6.png">

- Figma design:
<img width="301" alt="Screenshot 2022-03-20 at 1 03 54 AM" src="https://user-images.githubusercontent.com/70269507/159139961-d1456bfb-309c-4f8a-a559-b89851e8fddb.png">